### PR TITLE
Fixed incorrect No data in Grafana dashboard panels

### PR DESCRIPTION
**Pull request by msvticket**
Many panels show No data if Prometheus has a scrape interval configured to be larger than 15s. Using
the `$__rate_interval` introduced in Grafana 7.2 fixes just that.

- [ ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Dual Stack
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [x] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure
- [ ] Upgrade
- [ ] Multi Cluster
- [ ] Virtual Machine
- [ ] Control Plane Revisions

---

- [ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by jacob-delgado** [comment]
Do you have any screenshots?

/ok-to-test

File: releasenotes/notes/grafana-rate-interval.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- |
+  **Fixed** an issue where many panels in the Grafana dashboards showed **No data** if Prometheus had a scrape
+  interval configured to be larger than `15s`.
+
+# upgradeNotes is a markdown listing of any changes that will affect the upgrade
+# process. This will appear in the release notes.
+upgradeNotes:
+  - title: Grafana dashboard changes
+    content: |
+      The changes require varsion 7.2 or later of Grafana.
```
Comment (my-git9) [inline releasenotes/notes/grafana-rate-interval.yaml:41]: 'varsion' ---> 'version' ?

Comment (msvticket) [reply releasenotes/notes/grafana-rate-interval.yaml]: fixed

**Comment by msvticket** [comment]
> Do you have any screenshots?

No, I don't seem to have any saved. But basically it loks the same as in this issue istio/istio#13533

**Review by howardjohn — APPROVED** [review]

**Pull request merged**